### ISSUE TITLE
Variables name is pointer to another variable

### ISSUE THREAD

**Issue by dlsniper** [derekparker/delve#871]
1. What version of Delve are you using (`dlv version`)?
5390801
2. What version of Go are you using? (`go version`)?
go 1.8.3
3. What operating system and processor architecture are you using?
Windows 10, amd64

4. What did you do?
```go
package main

import "fmt"

func main() {
	a := [3]int{1, 2, 3}
	b := &a
	fmt.Println(b, *b) // set breakpoint here
}
```
5. What did you expect to see?
```
b = (*[3]int)(0xc04203c440)
*b = (*[3]int)(0xc04203c440)
```
6. What did you see instead?
```
b = (*[3]int)(0xc04203c440)
&a = (*[3]int)(0xc04203c440)
```

This can be confusing when debugging a more complex application.

Original issue in: https://youtrack.jetbrains.com/issue/GO-3165

**Comment by aarzilli** [comment]
> What did you expect to see?

Do you mean, that you want to see `a` and `b`? I don't see why `*b` should be there. It's not hard to do, just a call to maybeDereference in variablesByTag. The question is: should `a` get flagged somehow? It's like that because the compiler decided that it needs to escape to the heap (I think).

**Comment by aarzilli** [comment]
This was closed by commit 317ebe1c58d108c5983defdaeb14538e6f3fab00
